Composer/packages/adaptive-flow/src/adaptive-flow-editor/utils/cursorTracker/index.ts

```diff
@@ -15,7 +15,7 @@ export function moveCursor(
 ): { [key: string]: string | undefined } {
   const currentElement = selectableElements.find((element) => element.selectedId === id || element.focusedId === id);
   if (!currentElement) return { selected: id, focused: undefined };
-  let element: SelectorElement = currentElement;
+  let element: SelectorElement;
```

Comment:
will removing this initial assignment have any unwanted side effects?

Reply:
The switch statement below assigns it in the default case. This one was probably caught by the open source LGTM scan and is more of a lint issue than a security one I think.

Reply:
ah I didn't catch that, cool 👍